manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsApiClientImpl.java

```diff
@@ -86,6 +90,7 @@ public Connector createConnector(ConnectorRequest connectorRequest) {
         try {
             return connectorsAPI.createConnector(true, connectorRequest);
         } catch (ApiException e) {
+            LOGGER.error(e.getMessage());
```

Comment:
@manstis Let's be consistent in our approach please. So rather than `LOGGER.error` here (and in the `deleteConnector` method), can we take the same approach as we did in the `getConnector` workflow, so something like:

```
String message = String.format("Failed to create Connector of type '%s', in Connector Namespace '%s'. Received HTTP Response '%s'", connectorEntity.getConnectorType(), mcNamespace, e.getCode());
throw new ConnectorCreationException(message, e);
```

cheers,

Rob

Reply:
I'm not 100% convinced this will show the original cause in our error logging.

The implementation on `main` does `throw new ConnectorGetException("Error while retrieving the connector on MC Fleet Manager", e)` (which is comparable to your proposal) and on [MGDOBR-634](https://issues.redhat.com/browse/MGDOBR-634) we are not seeing the underlying cause being logged; which was the reason for @sutaakar reporting the issue.

Unless I am missing something I don't think your proposal guarantees the underlying cause is logged?

Reply:
Unsurprisingly I missed something stupid.